datadogclient/formatter_test.go

```diff
@@ -33,4 +33,16 @@ var _ = Describe("Formatter", func() {
 
 		Expect(result).To(HaveLen(1))
 	})
+
+	It("does not prepend prefix to `bosh.healthmonitor`", func() {
```

Comment:
should we not do this or potentially should we send a dupe metric with the newer name?